ci-operator/step-registry/baremetal/lab/post/provisioning-network/baremetal-lab-post-provisioning-network-commands.sh

```diff
@@ -13,7 +13,7 @@ fi
 # As the API_VIP is unique in the managed network and based on how it is reserved in the reservation steps,
 # we use the last part of it to define the VLAN ID.
 # TODO: find a similar unique value for dual stack and ipv6 single stack configurations?
-VLAN_ID=$(yq ".api_vip" "${SHARED_DIR}/vips.yaml")
+VLAN_ID=$(yq ".[0].ip" "${SHARED_DIR}/hosts.yaml")
```

Comment:
```suggestion
VLAN_ID=$(yq '.[] | select(.name|test("master-00")).ip' "${SHARED_DIR}/hosts.yaml")
```

let's ensure it's the first master.